pandas/core/computation/eval.py

```diff
@@ -156,9 +160,16 @@ def _check_for_locals(expr, stack_level, parser):
                 raise SyntaxError(msg)
 
 
-def eval(expr, parser='pandas', engine=None, truediv=True,
-         local_dict=None, global_dict=None, resolvers=(), level=0,
-         target=None, inplace=False):
+def eval(expr: str,
+         parser: str = 'pandas',
+         engine: str = None,
+         truediv: bool = True,
+         local_dict: dict = None,
```

Comment:
Instead of using `dict` can you use the generic `Dict` from the typing module and add subscripts for the key / value types?

Reply:
Thanks for this feedback. I looked through the where `local_dict` be used https://github.com/pandas-dev/pandas/blob/master/pandas/core/computation/scope.py#L103, but I cannot find out what kind of dict is this?  `[str, str]` ? Thanks !

Reply:
Hmm I'm not intimately familiar with this code but from what I gather should probably be `Dict[str, Any]` to simulate what can be injected into locals() and/or globals()

Reply:
Thanks!  I will update soon !